### DISCUSSION TITLE
Bugi-perjantaiden aloitus

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#806] [Tiedotteet]
Ideoimme alkuvuodesta, että alamme pitämään Koha-pääkäyttäjien kanssa Bugi-perjantaita, joissa edistämme meille tärkeitä ja kiinnostavia [Koha-yhteisön bugeja](https://bugs.koha-community.org/bugzilla3/) eli tikettejä.

Ihan ensimmäinen Bugi-perjantai pidetään perjantaina 13.10.2023 klo 13-15 [Jitsi Meetissä](https://meet.jit.si/adminsuomikoha). Toivomme työhön mukaan kaikkia Koha-Suomen kimppojen sekä tieteellisten Koha-kirjastojen pääkäyttäjiä. Työ organisoidaan julkisessa [Bugiton-repositoriossa](https://github.com/KohaSuomi/Bugiton).

Koha-Suomelta Anneli ja Emmi etsivät jo ennakkoon nipun bugeja tarkasteltavaksi, mutta osallistujat voivat myös itse etsiä sopivia bugeja.

Tervetuloa mukaan!

**Comment by AnneliO** [comment]
Bugi-perjantaita järjestetään noin kerran kuussa jatkossa.

**Comment by jmiettunen** [comment]
Yo, kiva tulla bugeja liiskaamaan.
